src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -383,11 +383,11 @@ public static class StandardTrainersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with <see cref="AveragedPerceptronTrainer"/>.
+        /// Create an <see cref="AveragedPerceptronTrainer"/>, which predicts a target using a linear binary classification model trained over boolean label data.
```

Comment:
>boolean label data [](start = 143, length = 18)

over a label of boolean data.

Reply:
Natalie voted for no change.

---
In reply to: [275001029](https://github.com/dotnet/machinelearning/pull/3310#discussion_r275001029) [](ancestors = 275001029)